swag_client/migrations/versions/v2.py

```diff
@@ -250,5 +251,6 @@ def downgrade(account):
     d_account['ours'] = True if account['owner'] == 'netflix' else False
     d_account['netflix'] = True if account['owner'] == 'netflix' else False
     d_account['services'] = v1_services
+    d_account['account_status'] = account['account_status']
```

Comment:
Do you really want to down add this to v1? It's not defined in the v1 schema.

Reply:
this was added to v1 schema